crates/fuel-core/src/service/genesis.rs

```diff
@@ -84,11 +85,11 @@ fn import_genesis_block(
     let database = database_transaction.as_mut();
     // Initialize the chain id and height.
 
-    let chain_config_hash = config.chain_conf.root()?.into();
-    let coins_root = init_coin_state(database, &config.chain_conf.initial_state)?.into();
-    let contracts_root =
-        init_contracts(database, &config.chain_conf.initial_state)?.into();
-    let messages_root = init_da_messages(database, &config.chain_conf.initial_state)?;
+    let height = config.chain_config.height;
+    let chain_config_hash = config.chain_config.root()?.into();
+    let coins_root = init_coin_state(database, &config.chain_state, height)?.into();
+    let contracts_root = init_contracts(database, &config.chain_state, height)?.into();
+    let messages_root = init_da_messages(database, &config.chain_state)?;
```

Comment:
Can we just have the `into` here instead of having a redundant variable right after?